moved/src/move_execution/tests/utils.rs

```diff
@@ -150,7 +155,7 @@ impl TestContext {
         let treasury_address = AccountAddress::ONE;
         let base_token = MovedBaseTokenAccounts::new(treasury_address);
         let mut transaction = TestTransaction::new(tx, tx_hash);
-        transaction.with_cost(l1_cost, base_token);
+        transaction.with_cost(l1_cost, l2_cost, base_token);
```

Comment:
The assertion below should be for the sum of L1 and L2 costs, since the cost can come from either one now:
```rust
assert_eq!(treasury_balance, U256::from(l1_cost + l2_cost));
```

Reply:
Good point! But actually I need to make it even smarter, as that wouldn't account for refunds. So I'm currently considering what is the most ergonomic way to refactor this assert.